src/components/layout/navigation.js

```diff
@@ -22,48 +22,15 @@ class Navigation extends Component {
     });
   }
 
-  handleMenuClick = e => {
-    const key = last(e.keyPath);
-    switch (key) {
-      case 'logout':
-        this.props.dispatch({ type: 'auth/logout' });
-        break;
-      case 'invoices':
-        this.setState({ openKeys: [], selectedMenuKeys: e.keyPath });
-        break;
-      case 'clients':
-        this.setState({ openKeys: [], selectedMenuKeys: e.keyPath });
-        break;
-      case 'settings':
-        if (e.keyPath.length > 1) {
-          this.setState({ selectedMenuKeys: e.keyPath });
-        } else {
-          const openKeys = this.state.openKeys.length === 0 ? ['settings'] : [];
-          this.setState({ openKeys });
-        }
-        break;
-      default:
-        this.setState({ selectedMenuKeys: e.keyPath });
-    }
-  };
-
   render() {
-    const { selectedMenuKeys, openKeys } = this.state;
-
     return (
       <Layout.Sider trigger={null} collapsible collapsed={this.props.collapsed}>
         <div className="logo" style={{ height: 22, margin: '21px 16px', textAlign: 'center' }}>
           <Link to="/invoices/" onClick={() => this.setState({ selectedMenuKeys: ['invoices'] })}>
             <img src={require(`../../assets/logo.svg`)} alt="Upcount" />
           </Link>
         </div>
-        <Menu
-          theme="dark"
-          mode="inline"
-          selectedKeys={selectedMenuKeys}
-          openKeys={openKeys}
-          onClick={this.handleMenuClick}
-        >
+        <Menu theme="dark" mode="inline" defaultSelectedKeys={['1']}>
```

Comment:
There is no key with string '1' so the defaultSelectedKeys could be 'invoices'.

Reply:
ja du hast. Behebe ich mit dem nächsten Request

Reply:
fixed by selecting from localStorage